compliance/index/v4/convert_test.go

```diff
@@ -0,0 +1,309 @@
+package v4
+
+import (
+	"testing"
+
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoassert"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestIndexReportConvert(t *testing.T) {
+	suite.Run(t, new(indexReportConvertSuite))
+}
+
+type indexReportConvertSuite struct {
+	suite.Suite
+}
+
+func (s *indexReportConvertSuite) TestToNodeInventory() {
+	r := createVulnerabilityReport()
+
+	actual := ToNodeScan(r)
+
+	s.Equal(storage.NodeScan_SCANNER_V4, actual.GetScannerVersion())
+	s.Equal(1, len(actual.GetComponents()))
```

Comment:
```suggestion
	s.Len(actual.GetComponents(), 1)
```

Reply:
Absolutely right, thanks for paying attention